### PR TITLE
Update randomization.feature

### PR DESCRIPTION
**Pull request by cadamini**
This pull request tries to update the randomization texts for more readability and clarity.

- I have tried to parallelize the list in this doc. 
- I have tried to simplify most of the intro texts.

**Note:** Still unsure if I completely understood all concepts correctly. Please double-check if my updated information is correct. Especially lines 9-11 and 17 needs a real review :)

### CODE REVIEW

File: features/command_line/randomization.feature [hunk 1]
```diff
@@ -1,37 +1,28 @@
 Feature: Randomization can be reproduced across test runs
 
-  In Ruby, randomness is seeded by calling `srand` and passing it the seed that
-  you want to use. By doing this, subsequent calls to `rand`, `shuffle`,
-  `sample`, etc. will all be randomized the same way given the same seed is
-  passed to `srand`.
+  In Ruby, you can call `srand` for randomness and pass it to the seed you want to use. All subsequent calls to `rand`, `shuffle`,
+  `sample` all be randomized the same way.
```
Comment (cadamini) [inline features/command_line/randomization.feature:4]:
```suggestion
  In Ruby, you can call `srand` for randomness and pass it to the seed you want to use. All subsequent calls to `rand`, `shuffle`, or `sample` are randomized the same way.
```

File: features/command_line/randomization.feature [hunk 2]
```diff
@@ -1,37 +1,28 @@
...
 
   RSpec takes care not to seed randomization directly when taking action that
-  involves randomness (such as random ordering of examples).
+  involves randomness. 
 
-  Since RSpec does not ever invoke `srand`, this means that you are free to
-  choose which, if any, mechanism is used to seed randomization.
+  RSpec do not trigger randomization directly for actions that involve randomness.
```
Comment (cadamini) [inline features/command_line/randomization.feature:9]: unclear or vague. consider removing this.

File: features/command_line/randomization.feature [hunk 3]
```diff
@@ -1,37 +1,28 @@
 Feature: Randomization can be reproduced across test runs
 
-  In Ruby, randomness is seeded by calling `srand` and passing it the seed that
-  you want to use. By doing this, subsequent calls to `rand`, `shuffle`,
-  `sample`, etc. will all be randomized the same way given the same seed is
-  passed to `srand`.
+  In Ruby, you can call `srand` for randomness and pass it to the seed you want to use. All subsequent calls to `rand`, `shuffle`,
```
Comment (pirj) [inline features/command_line/randomization.feature:3]: > call `srand` for randomness

https://ruby-doc.org/core-2.7.1/Random.html#method-c-srand

> Seeds the system pseudo-random number generator


> `srand` for randomness and pass it to the seed you want to use

This is confusing. Pass what to the seed? How can you pass anything to a number which a seed is?


I believe the wording before this change is more correct.

File: features/command_line/randomization.feature [hunk 4]
```diff
@@ -1,37 +1,28 @@
...
-  If you would like to manage seeding randomization without any help from RSpec,
-  please keep the following things in mind:
+    * Do not hard-code the seed.
 
-    * The seed should never be hard-coded.
-
-      The first example below only does this to show that seeding randomization
-      with a seed other than the one used by RSpec will correctly seed
-      randomization.
+      You can still seed the correct randomization with a seed other than the one used by RSpec.
```
Comment (pirj) [inline features/command_line/randomization.feature:17]: Can you please elaborate on this?

**Comment by pirj** [comment]
Sorry, those changes don't seem to be a reasonable improvement to our current documentation.

**Pull request closed without merging**